resources/views/projects/entity-step/4.blade.php

```diff
@@ -11,28 +11,28 @@
         <p class="align-end"><a href="{{ localized_route('collections.index') }}">{{ __('Find access providers in the Resource Hub') }}</a></p>
     </div>
     @endif
-    @if(count($project->confirmedConsultants) > 0)
-    <h3>{{ __('Consultants') }}</h3>
-    @foreach($project->confirmedConsultants as $consultant)
-    <x-consultant-card :consultant="$consultant" level="4">
-        <x-heading :level="5">{!! __('<span class="visually-hidden">:name’s </span>Contact information', ['name' => $consultant->name]) !!}</x-heading>
+    @if(count($project->confirmedParticipants) > 0)
+    <h3>{{ __('Participants') }}</h3>
+    @foreach($project->confirmedParticipants as $communityMember)
```

Comment:
Are you using `$communityMember` here instead of `$participant` to identity what "type" is being acted upon?

Reply:
Yep.